scripts/overpass-way.php

```diff
@@ -43,7 +43,8 @@ function get(string $city): string
     $client = new \GuzzleHttp\Client();
     $response = $client->request(
         'GET',
-        sprintf('https://overpass-api.de/api/interpreter?data=%s', urlencode($query))
+        sprintf('https://overpass-api.de/api/interpreter?data=%s', urlencode($query)),
+        ['timeout' => 400]
```

Comment:
This is not necessary !
The default Guzzle behavior is no timeout (see https://docs.guzzlephp.org/en/stable/request-options.html#timeout)
The timeout for the Overpass query should only be defined in the 2 overpass query files.